### PR TITLE
Feature/api spec

### PR DESCRIPTION
**Pull request by ltankey**
Initial draft spec and example API

Open for mass-disucssion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # GOV.AU Beta Content Analysis
 This project provides the content analysis API for the GOV.AU content site.
 
+The API Specification draft can be found [here](./api-spec.md)
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
```
Comment (tessereth) [inline README.md]: We probably don't want this default README scaffolding back in again.

File: api-spec.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    /api/linters
+  
+* **Method**
+  
+    `POST`
+ 
+* **Parameters**
+
+    HEAD: `content=[String]`
```
Comment (tessereth) [inline api-spec.md]: Doesn't HEAD imply header parameters? Aren't these post parameters?

Comment (ltankey) [inline api-spec.md]: Potentially. You could also choose to map out URL parameters if you wanted. Would 

`POST: content=[String]` be more informative?

Comment (tessereth) [inline api-spec.md]: I think so.

**Comment by tessereth** [comment]
LGTM. I might leave the merge until others have had a chance to look at it though.

**Pull request merged**